File: 3859-maximum-product-of-two-digits/maximum-product-of-two-digits.py
class Solution(object):
    def maxProduct(self, n):
        s=1
        k=[]
        while n>0:
            r=n%10
            k.append(r)
            n=n/10
        k=sorted(k)
        n=len(k)
        return k[n-1]*k[n-2]
        

        

File: 3859-maximum-product-of-two-digits/maximum-product-of-two-digits.java
import java.util.*;

class Solution {
    public int maxProduct(int n) {
        ArrayList<Integer>list=new ArrayList<>();
        while(n>0){
           int r=n%10;
           list.add(r);
           n=n/10;
        }
        Collections.sort(list);
        int l=list.size();
        int result=list.get(l-1)*list.get(l-2);
        return(result);
    }
    
}